smart_card_connector_app/src/background.js

```diff
@@ -165,6 +166,28 @@ if (GSC.ExecutableModule.TOOLCHAIN ===
   GSC.BackgroundPageUnloadPreventing.enable();
 }
 
+if (chrome.smartCardProviderPrivate !== undefined) {
+  const HANDLE_CHROME_REQUESTS = 'handle_requests_from_chrome_smart_card_api';
+  chrome.storage.managed.get(HANDLE_CHROME_REQUESTS, (policies) => {
```

Comment:
Does Chrome expect the event listeners to be subscribed immediately on the app's startup?

I'm asking because this `chrome.storage.managed.get()` call introduces some asynchronous delay. I haven't checked how Chrome side is implemented - whether it waits for the app's installation, startup or the event listener subscription.

Reply:
if no one is subscribed to the events yet, it will return related error to all requests coming from the browser.

Reply:
Ack. Might be fine, as long as the application that consumes the Web API is aware of this behavior.